app/queries/get_activity_log_events.rb

```diff
@@ -52,6 +54,15 @@ def self.application_choice_audits_filter_sql
     filtered_statuses = ApplicationStateChange::STATES_VISIBLE_TO_PROVIDER - IGNORE_STATUS
     status_transitions_to_include = filtered_statuses.map { |status| "'#{status}'" }.join(', ')
 
-    application_choice_audits_filter + " OR audited_changes::json#>>'{status, 1}' IN (#{status_transitions_to_include})"
+    application_choice_audits_filter += " OR audited_changes::json#>>'{status, 1}' IN (#{status_transitions_to_include})"
+    application_choice_audits_filter + interview_cancelled_application_choice_status_change_sql
+  end
+
+  def self.interview_cancelled_application_choice_status_change_sql
```

Comment:
```suggestion
  def self.ignore_interview_cancelled_application_choice_status_change_sql
```
just as a form of documentation